### PR TITLE
[ci skip] Update guide sources for mailer view caching feature

### PR DESCRIPTION
**Pull request by st0012**

### CODE REVIEW

**Comment by st0012** [comment]
cc @rafaelfranca

File: guides/source/5_0_release_notes.md [hunk 1]
```diff
@@ -316,6 +316,9 @@ Please refer to the [Changelog][action-mailer] for detailed changes.
 *   Template lookup now respects default locale and I18n fallbacks.
     ([commit](https://github.com/rails/rails/commit/ecb1981b))
 
+*   Template can use fragment cache like ActionView template.
```
Comment (rafaelfranca) [inline guides/source/5_0_release_notes.md]: `Action View`

File: guides/source/action_view_overview.md [hunk 2]
```diff
@@ -599,7 +599,7 @@ This would add something like "Process data files (0.34523)" to the log, which y
 
 #### cache
 
-A method for caching fragments of a view rather than an entire action or page. This technique is useful for caching pieces like menus, lists of news topics, static HTML fragments, and so on. This method takes a block that contains the content you wish to cache. See `ActionController::Caching::Fragments` for more information.
+A method for caching fragments of a view rather than an entire action or page. This technique is useful for caching pieces like menus, lists of news topics, static HTML fragments, and so on. This method takes a block that contains the content you wish to cache. See `ActionDispatch::Caching::Fragments` for more information.
```
Comment (rafaelfranca) [inline guides/source/action_view_overview.md]: `AbstractController`

**Comment by st0012** [comment]
@rafaelfranca fixed

**Comment by st0012** [comment]
@rafaelfranca thanks!

**Pull request merged**